keyboards/1upkeyboards/1up60hse/config.h

```diff
@@ -112,10 +112,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
 
-/* key combination for magic key command */
-#define IS_COMMAND() ( \
-    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
-)
 
```

Comment:
I suggest removing the extra newlines as well so we don't end up with lots of files with double empty lines.

Here's a shell script that will find and replace these extra newlines in the files changed by this PR:
```sh
for file in $(git diff --numstat master...HEAD | awk '{print $3}'); do
        tmp="$file.tmp"
        tr '\n' '\f' < "$file" | sed -e 's/\f\f\f/\f\f/g' | tr '\f' '\n' > "$tmp" && mv "$tmp" "$file" &
done
```